coalib/bearlib/languages/definitions/Haskell.py

```diff
@@ -0,0 +1,20 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class Haskell:
+    extensions = '.hs', '.lhs',
```

Comment:
you do analysis before blindly adding something because of review comments.

Reply:
> you do analysis before blindly adding something because of review comments.

I clearly give a question mark in the previous review tho.